src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/emails/StaffEmailTemplates.java

```diff
@@ -42,6 +42,10 @@ public String getCCJRequestSubmittedEmailSubject() {
         return readBytes("/staff/templates/document/legalSealedClaim.html");
     }
 
+    public byte[] getWaterMarkedLegalSealedClaim() {
+        return readBytes("/staff/templates/document/waterMarkedLegalSealedClaim.html");
```

Comment:
Is it not possible to move this check to inside the html file so that the water mark only gets added if on demo? other wise we will have to maintain two template files when making changes